src/dune_rules/lib.mli

```diff
@@ -41,6 +41,13 @@ val hash : t -> int
 
 val project : t -> Dune_project.t option
 
+module Kind_db : sig
+  type t =
+    | Installed_libs
+    | Public_libs
+    | Project_libs
```

Comment:
is the distinction between project and public libs used? I wonder if we could do without this `kind` field. we can check if a db is the installed_lib ones by checking if its parent is None maybe?